data/hai/hai reveal 5 philosophers.txt

```diff
@@ -848,18 +943,44 @@ mission "Hai Reveal [B12] Devil-Hide Battle"
+			`	"Of course, nobody tells me anything in a timely manner..."`
+			`	"Anyway, I've organized the Syndicate's 'hostile urban insertion security operatives' to be loaded onto the Auxiliary. They look like intelligence operatives, if you ask me, so that'll be a conversation for me to follow up on later, but for now they're specially trained for attacking pirate strongholds and equipped with state of the art equipment that... is on a need-to-know basis. As long as we can protect the Auxiliary we'll be fine. The intel you brought back doesn't suggest particularly strong resistance. Wallace, good luck."`
+			choice
+				`	"It's good to hear you have the ground combat situation under control."`
+					goto control
+				`	"That equipment sounds ominous."`
+			`	Kopkow gives you a disturbing smile that has an 'I love my work' appearance. He departs as everyone else assembles.`
+				goto leave
+			label control
+			`	"It's more of a collaboration, really, but like I said: need-to-know basis." He nods to you and departs as everyone else assembles.`
+			label leave
+			`	After a large meeting about logistics, and an unsurmountable buffet, everyone disperses. General Choot'k finds you to discuss critical tactical matters.`
+			`	"You have handled yourself well throughout this. I want to see you prove yourself in battle before I open the entrance to <planet>. When we arrive at <planet>, we'll first fight the defense fleet waiting for us. I suspect we can handle it, but you are of course welcome to assist.`
+			`	"The main priority for you should be to make sure nobody pursues Elder Teeneep or Captain Wallace. However, these scum will likely keep a couple of ships in reserve. The scans you took suggest some are not totally fighting fit, or perhaps will be kept to use their weapons inside the spaceport as a backup defense.`
+			`	"To counter this, you'll need to draw them out by demanding tribute from the city. They'll keep sending defense fleets until they resign to their fate and pay you daily tribute. This is where we would like you to prove yourself. We won't help you fight the tribute defense fleets, but we'll fight the pirates waiting for our arrival and any other pirates that show up."`
```

Comment:
I wasnt suggesting a workaround, since its not possible.
I was just suggesting that, since its an engine limitation, we find a reason for it lore wise or it'll be a plothole the way I see it. And isnt good gameplay.
If you prepare for help from 2 heavy warships and then arrive on the battle and find that you're alone...

We won't have anything to make them attack tribute fleets anytime soon. What we could do is change the whole mission to make you land and spawn the ships at that moment but that wouldn't be the same thing anymore, I don't think we should change that.

So maybe just say smth like "the humans will focus on defeating incoming ships so that you can take care of the defense fleet"? Or they will cover the Unfettered ships? Idk what we need to do, but we need to do something so it isnt this weird.
Like at the very least a big warning "you'll have to fight a consequential amount of warships alone"
```suggestion
			`	"To counter this, you'll need to draw them out by demanding tribute from the city. They'll keep sending defense fleets until they resign to their fate and pay you daily tribute. This is where we would like you to prove yourself. We won't help you fight the tribute defense fleets, but we'll fight the pirates waiting for our arrival and any other pirates that show up. Given the consequential amount of warships we expect to see, I estimate the humans may have to support us in combat, which would mean you truly will have to handle the tribute fleet alone. Prepare adequately, captain."`
```
Here, this is what I mean, although you don't have to apply this exact change.

Reply:
It is a good approach. I will ponder the correct wording and change this soon.

> He takes you apart to warn you

Tearing Captain Last's body apart seems unnecessarily violent for a warning about combat tactics, even for a squirrel.

Reply:
you made me laugh x)
anyway idk much about english in case it wasnt obvious enough, but I tried to make this as good as I could

Reply:
You've come close to the right text. What matters the most is presenting a tribute demand as a reasonable battle tactic, which I think you've done well. I just need to tweak the wording.

Reply:
This whole thing is weird to me
Where do the thousand ships coming in (after some time) come from even? Deep missions? where they went?

Reply:
You reduced that problem considerably in your PR to balance the Devil-Hide battle. 

The remaining problem is the constant inflow of capital ships. They do need some because, according to the lore, there are dangerous apex predators in the Whispering Void. Probably not a massive armada, though. There's also supposed to be a shipyard, though it shouldn't be able to make new ships like Falcons, or specialized ships like Bactrians.

Reply:
True, my fleet having more firebirds that they could produce there would help a bit. If there are dangerous aliens here and the Unfettered notice them thats perfect for smth I have planned for the UHai story. Could you send me the image of those wales?

Reply:
They can also make smaller, low-tech, Hai outfits. Hybrid human-Hai Firebird and Leviathan builds may let us have fewer ships with a similar difficulty level.

Reply:
We can do that once the balancing is merged, to make it better lore wise

Reply:
It was a bit of work, but I think I have a good justification for the tribute demand now. Please look at it and let me know what you think.

Reply:
hybrid ships are a neat idea for a future PR <3